### ISSUE TITLE
Ikoner har elementer med duplikate id-er

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-enonicxp-frontend#1422]
https://my2.siteimprove.com/Accessibility/1002489/NextGen/Issue/1?ruleId=3&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&siteTarget.conformanceLevels=0,1,2,3,4&siteTarget.issueKinds=1,2

Kan man fjerne id-er når man generer svg-ene?

**Comment by PerOlavM** [comment]
Diskutert på Monday Commit.
Chris tar en sjekk ifm. arbeidsgiversidedene. Kan svg-er eksporteres bedre?
Vurderer ev. tiltak senere

**Comment by chrisarnesen** [comment]
Sendt test ikon på Slack, med andre Figma innstillinger, løser det problemet?

**Comment by PerOlavM** [comment]
Problemet er piktogrammene hvor vi har lastet inn svg-er fra Figma manuelt.
Her er det ikke ryddet opp ved eksport. Jobben blir derfor manuelt renske opp aktuelle svg-er.

**Comment by PerOlavM** [comment]
Gått igjennom alle 153 piktogrammene og oppdatert alle som har id i svg-koden
